presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -194,16 +202,18 @@ public void testCosh()
             assertFunction("cosh(FLOAT '" + (float) doubleValue + "')", DOUBLE, Math.cosh((float) doubleValue));
         }
         assertFunction("cosh(NULL)", DOUBLE, null);
+        assertFunction("cosh(CAST(NULL AS FLOAT))", DOUBLE, null);
     }
 
     @Test
     public void testDegrees()
     {
         for (double doubleValue : DOUBLE_VALUES) {
             assertFunction(String.format("degrees(%s)", doubleValue), DOUBLE, Math.toDegrees(doubleValue));
-            assertFunction(String.format("degrees(FLOAT '%s')", (float) doubleValue), DOUBLE, Math.toDegrees((float) doubleValue));
+            assertFunction(String.format("degrees(FLOAT '%s')", doubleValue), DOUBLE, Math.toDegrees((float) doubleValue));
```

Comment:
I believe `(float) doubleValue` should be here to ensure that we build FLOAT's string out of float value instead of relying on implicit casting of FLOAT c-tor (`Float.parseString`).